grpc_api/descriptor_record_store_pb.bal

```diff
@@ -0,0 +1,166 @@
+        var payload = check self.grpcClient->executeServerStreaming("Albums/listAlbums", message, headers);
+        [stream<anydata, grpc:Error?>, map<string|string[]>] [result, _] = payload;
+        AlbumStream outputStream = new AlbumStream(result);
+        return new stream<Album, grpc:Error?>(outputStream);
+    }
+
+    isolated remote function listAlbumsContext() returns ContextAlbumStream|grpc:Error {
+        empty:Empty message = {};
+        map<string|string[]> headers = {};
+        var payload = check self.grpcClient->executeServerStreaming("Albums/listAlbums", message, headers);
+        [stream<anydata, grpc:Error?>, map<string|string[]>] [result, respHeaders] = payload;
+        AlbumStream outputStream = new AlbumStream(result);
+        return {content: new stream<Album, grpc:Error?>(outputStream), headers: respHeaders};
+    }
+}
+
+public class AlbumStream {
+    private stream<anydata, grpc:Error?> anydataStream;
+
+    public isolated function init(stream<anydata, grpc:Error?> anydataStream) {
+        self.anydataStream = anydataStream;
+    }
+
+    public isolated function next() returns record {|Album value;|}|grpc:Error? {
+        var streamValue = self.anydataStream.next();
+        if (streamValue is ()) {
+            return streamValue;
+        } else if (streamValue is grpc:Error) {
+            return streamValue;
+        } else {
+            record {|Album value;|} nextRecord = {value: <Album>streamValue.value};
+            return nextRecord;
+        }
+    }
+
+    public isolated function close() returns grpc:Error? {
+        return self.anydataStream.close();
+    }
+}
+
+public client class AlbumsAlbumCaller {
+    private grpc:Caller caller;
+
+    public isolated function init(grpc:Caller caller) {
+        self.caller = caller;
+    }
+
+    public isolated function getId() returns int {
+        return self.caller.getId();
+    }
+
+    isolated remote function sendAlbum(Album response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendContextAlbum(ContextAlbum response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendError(grpc:Error response) returns grpc:Error? {
+        return self.caller->sendError(response);
+    }
+
+    isolated remote function complete() returns grpc:Error? {
+        return self.caller->complete();
+    }
+
+    public isolated function isCancelled() returns boolean {
+        return self.caller.isCancelled();
+    }
+}
+
+public type ContextAlbumStream record {|
+    stream<Album, error?> content;
+    map<string|string[]> headers;
+|};
+
+public type ContextAlbum record {|
+    Album content;
+    map<string|string[]> headers;
+|};
+
+@protobuf:Descriptor {value: DESCRIPTOR_RECORD_STORE_DESC}
+public type Album record {|
+    string id = "";
+    string title = "";
+    string artist = "";
+    float price = 0.0;
+|};
+
```

Comment:
Extra new line.

Reply:
Seems like additional line is added during grpc generation. Added an issue to track this from[ grpc side](https://github.com/ballerina-platform/ballerina-standard-library/issues/3953).

Since regenration will include this again will include this line shall we leave it like that? Anyway this file should not be committed once GH Action issue is ficed